win or loss.py

```diff
@@ -0,0 +1,35 @@
+# FOR ATLANTA
+
+homeTeam = homeTeamATL
+oppTeam = awayTeamATL
+
+
+def CallWinner():
+    if str(homeScoreATL) > str(awayScoreATL):
+        return homeTeam
+
+
+winner = CallWinner()
```

Comment:
So THIS actually gives you the win or loss?

Reply:
Yes because the two functions at the bottom actually differentiate 
based on the string comparisons

So it looks to see if the winner matches the focusTeam 
which I just determined by entering the string of the
'focus team' comparing it to the winner

Reply:
Oh nice, I get it